molecule/default/prepare.yml

```diff
@@ -2,4 +2,8 @@
 - name: Prepare
   hosts: all
   gather_facts: false
-  tasks: []
+  tasks:
+    - name: Install rsync for grafana dashboard provisioning
+      package:
+        name: ["rsync"]
+      when: grafana_use_provisioning
```

Comment:
`grafana_use_provisioning` can be undefined when using here, just delete this check and it is ready to go.